### DISCUSSION TITLE
Trying to Install SAP NWRFC SDK Installation and Enable pyrfc

### DISCUSSION THREAD

**Discussion by AFK-Python** [SAP/PyRFC#347] [Q&A] [answered]
Hi there,

Apologies in advance if these questions are silly, I am not very familiar with any of the tooling here.

My first question is regarding the SAP NWRFC SDK Installation. I downloaded the zip file, `nwrfc750P_12-70002726.zip`, but after following further instructions underneath the Mac install section, I can't seem to get `./startrfc -v` working, and `from pyrfc import *` also doesn't work.

A couple of questions here:

1. Do I also need to download the hotfix (`nwrfc750P12HF_2-70002726.zip`)? It is much smaller than the zip I downloaded... so is it to be used in conjunction with the zip I have?
2. What happened to the `paths_fix.sh` script? The link does not seem to be working. What do I need to do for that part of the installation.

Any help would be greatly appreciated!

**Comment by bsrdjan** [comment]
No silly questions, you are on the right path but the link in [SAP NWRFC SDK installation documentation](https://sap.github.io/PyRFC/install.html#sap-nwrfc-sdk-installation) was broken, after moving the script to  [SAP/fundamental-tools](https://github.com/SAP/fundamental-tools) repository.

It is fixed now and here the correct url [paths_fix.sh](https://github.com/SAP/fundamental-tools/blob/main/docker/sap_nwrfcsdk_utils/paths_fix.sh)

I just checked the SAP Support Portal and SAP NW RFC SDK for Mac starts with nwrfc750P_12-8* not 7*. Could it be you downloaded RFC SDK for another platform?

After RFC SDK for your Mac (Intel or ARM) is downloaded, you can run the patch script and use the RFC SDK and PyRFC.

**Reply by AFK-Python** [reply]
Thanks for the quick reply!

I am able to see the script now, I appreciate that.

It is definitely possible that I accidentally downloaded the wrong RFC SDK. Do you happen to have a link for the Mac version of the SDK? I've been navigating the [downloads](https://me.sap.com/swdcnav/products/_APP=00200682500000001943&_EVENT=DISPHIER&HEADER=Y&FUNCTIONBAR=N&EVENT=TREE&NE=NAVIGATE&ENR=01200314690100002214&V=MAINT) (I hope that url works) and only see the versions I mentioned in the original post. I can't seem to find one that has the 8 like you mentioned.

**Reply by bsrdjan** [reply]
The url works. Can you send the screenshot showing the versions you see?

**Reply by AFK-Python** [reply]
Yes! Here it is:

<img width="1792" alt="Screenshot 2023-12-01 at 10 02 13 AM" src="https://github.com/SAP/PyRFC/assets/54761070/db72a082-526c-449f-8bc6-50905dd0d408">

**Reply by bsrdjan** [reply]
Open the drop down showing "AIX 64BIT"

**Reply by AFK-Python** [reply, marked as answer]
🤦 Can't believe I missed that.

Thanks for the catch!

**Reply by HalukUckurt** [reply]
Hello, 
Do you still have the file? I'm having problems with the higher patch levels. It would be great if I can get it.
Thank you
